src/pages/Login.js

```diff
@@ -0,0 +1,149 @@
+import axios from "axios";
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { useEffect } from "react";
+import "../components/css/login.css";
+import { notificationHandler } from "../components/Notification";
+import { useUserContext } from "../context & data/UserContext";
+
+const LoginPage = () => {
+  const [action, setAction] = useState("login");
+  const navigate = useNavigate();
+  const { dispatch } = useUserContext();
+
+  useEffect(() => {
+    if (
+      localStorage.getItem("isAuth") &&
+      localStorage.getItem("isAuth") !== "false" &&
+      localStorage.getItem("token").length > 25
+    )
+      navigate(-1);
+  }, [navigate]);
+
+  const submitHandler = (e) => {
+    e.preventDefault();
+    const formData = new FormData(e.target);
+    let data = {};
+    for (let x of formData) {
+      data[x[0]] = x[1];
+    }
+    axios
```

Comment:
You can try async/await instead of .then. That's more handy for error handling too.

Reply:
right will give it a try.